docs/resources/Application.md

```diff
@@ -30,8 +30,8 @@
 | flags?                             | integer                                                                                                                               | App's public [flags](#DOCS_RESOURCES_APPLICATION/application-object-application-flags)                                                                                                                                                     |
 | approximate_guild_count?           | integer                                                                                                                               | Approximate count of guilds the app has been added to                                                                                                                                                                                      |
 | redirect_uris?                     | array of strings                                                                                                                      | Array of redirect URIs for the app                                                                                                                                                                                                         |
-| interactions_endpoint_url?         | string                                                                                                                                | [Interactions endpoint URL](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/receiving-an-interaction) for the app                                                                                                                              |
-| role_connections_verification_url? | string                                                                                                                                | Role connection verification URL for the app                                                                                                                                                                                               |
+| interactions_endpoint_url?         | ?string                                                                                                                               | [Interactions endpoint URL](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/receiving-an-interaction) for the app                                                                                                                              |
```

Comment:
Thanks for the change here, this matches the potential behavior

Reply:
Then the field can be null and not be present?

In what cases can not be present? Just curious.

Reply:
While less than idea, that is correct.  I think in most areas of our API it's going to just be missing from the payload if it's not set.  In this particular case, for whatever reason, we tend to include and make it nullable.  I want to reserve the right to make it optional in the payload like other fields, so documenting it as potentially not there and potentially null gives us the most flexibility.